dapp_stats/dapp_stats.py

```diff
@@ -0,0 +1,52 @@
+from collections import defaultdict
+import pydantic
+from typing import DefaultDict, Dict, Optional
+
+from dapp_manager import DappManager
+
+from .exceptions import DappStatsException
+from .statistics.models import NodeStat
+from .statistics.schemas import StateLogEntry
+
+
+class DappStats:
+    def __init__(self, app_id: str):
+        self._app_id = app_id
+
+    def _iter_app_states(self):
+        return DappManager(self._app_id).storage.iter_file("state")
+
+    def get_stats(self) -> Dict:
+        nodes_stats: DefaultDict[str, Dict[int, NodeStat]] = defaultdict(dict)
+        app_stats: Optional[NodeStat] = None
+
+        for raw_state in self._iter_app_states():
+            try:
+                app_state = StateLogEntry.parse_raw(raw_state)
```

Comment:
again, I'm finding the difference between `app_stats` and `app_state` a bit too small to make reading the code comfortable... might I suggest `app_statistics` instead?